src/components/AppMainView.tsx

```diff
@@ -46,25 +44,23 @@ const AppMainView = () => {
             <WarningView />
             <div className="memory-box-container">
                 <MemoryBoxView
-                    title="File memory layout"
-                    description="Drag & drop HEX/ZIP files here"
-                    iconName="mdi mdi-folder-open"
+                    title="Memory visualization"
+                    description="Drag & drop HEX/ZIP/ELF files here"
+                    iconName="mdi mdi-file-outline"
                     isHolder={!hasFileContent(loaded) && !zipFilePath}
+                    hasViz={elf !== undefined}
                 />
                 <MemoryBoxView
                     title={
                         getTargetTitle(serialNumber, deviceInfo) ||
                         'Device memory layout'
                     }
                     description="Connect a device to display memory contents"
-                    iconName="appicon-chip"
+                    iconName="mdi appicon-chip"
```

Comment:
Is mdi added here for some kind of layout css?

Reply:
![image](https://user-images.githubusercontent.com/55808963/187428035-dde69ae3-ea90-4e1d-b9c9-f1d72ec7d8c0.png)
Yep, and I'm assuming that this change was made to inherit the correct fonts since that's what I changed for the "icon with file extension" feature that we discussed above.